frontend/src/components/navbar.tsx

```
@@ -0,0 +1,156 @@
+import React from "react";
+import { Kelly_Slab } from "next/font/google";
+const kelly = Kelly_Slab({
+  weight: ["400"],
+  subsets: ["latin"],
+});
+const navbar = () => {
+  return (
+    <div className="navbar">
+      <div className="navbar-start ml-[-24px]">
+        <div className="dropdown ">
+          <div tabIndex={0} role="button" className="btn btn-ghost lg:hidden">
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)